frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/OnboardingModal.tsx

```diff
@@ -0,0 +1,135 @@
+
+export interface IOnboardingModalProps {
+    close: () => void
+    open: boolean
+}
+
+export function OnboardingModal(props: IOnboardingModalProps) {
+    const { t } = useTranslation()
+
+    const cards = useMemo(
+        () => [
+            {
+                id: 'onboardingimportbutton',
+                icon: <ImportIcon />,
+                text: t('Import an existing cluster'),
+                link: NavigationPath.importCluster,
+            },
+            {
+                id: 'onboardingconnectbutton',
+                icon: <CloudTenantIcon />,
+                text: t('Connect your cloud provider'),
+                link: NavigationPath.addCredentials,
+            },
+            {
+                id: 'onboardingdiscoverbutton',
+                icon: <ConnectedIcon />,
+                text: t('Discover hosts to create host inventory'),
+                link: NavigationPath.discoveredClusters,
+            },
+        ],
+        [t]
+    )
+    if (props.open === false) {
+        return <></>
+    }
+
+    return (
+        <AcmModal
+            variant={ModalVariant.medium}
+            title=" "
+            isOpen={true}
+            onClose={props.close}
+            footer={
+                <AcmExpandableSection
+                    label={t('Want to learn more?')}
+                    style={{
+                        backgroundColor: '#F0F0F0',
+                        width: '100%',
+                        height: '100%',
+                        padding: '16px 16px 16px 36px',
+                    }}
+                >
+                    <div>
+                        <Link key="gettingstarted" to={`${NavigationPath.managedClusters}?quickstart=add-healthchecks`}>
+                            {t('Getting Started with on-premise host inventory')}
+                        </Link>
+                    </div>
+                    <div>
+                        <Text
+                            component={TextVariants.a}
+                            isVisitedLink
+                            href={DOC_BASE_PATH}
+                            target="_blank"
+                            style={{
+                                cursor: 'pointer',
+                                display: 'inline-block',
+                            }}
+                        >
+                            {t('Learn more about Advanced Cluster Management')} <ExternalLinkAltIcon />
+                        </Text>
+                    </div>
+                </AcmExpandableSection>
+            }
+        >
+            <Grid>
+                <GridItem span={3}>
+                    <MainIcon />
+                </GridItem>
+                <GridItem span={9}>
+                    <div>
+                        <span style={{ color: '#393F44', fontSize: '24px' }}>
+                            {t('Managing clusters')}{' '}
+                            <span className="pf-u-font-weight-bold">{t('just got easier')}</span>
```

Comment:
I recommend you use the `<Trans>` component here instead of separate strings.

Reply:
Yep I'll make that change. Thanks @KevinFCormier